config/target.json

```diff
@@ -0,0 +1,5 @@
+{
+  "generate_icicle": false,
+  "openstack_image_format": "qcow2",
+  "rhevm_image_format": "qcow2"
```

Comment:
I tried it out.  For both RHEVM and OpenStack, it didn't go through the conversion unless I put the settings back to imagefactory.conf.